README.md

```diff
@@ -50,4 +50,21 @@ Currently there's no vendoring provided for this project.
 * **vulcan-unclassified** - Example vulnerability to test the monitoring of unclassified vulnerabilities - not for production
 * **vulcan-vulners** - Runs https://vulners.com/api/v3/burp/software/
 * **vulcan-wpscan** - Checks Wordpress sites for vulnerabilities using the open source wpscan utility
-* **vulcan-zap** - Checks for vulnerabilities in web applications using OWASP ZAP
\ No newline at end of file
+* **vulcan-zap** - Checks for vulnerabilities in web applications using OWASP ZAP
+
+## Building and testing
+
+This project is primarily built using the [vulcan-checks-bsys](https://github.com/adevinta/vulcan-checks-bsys) project.
+
+But it's possible to build each one of the checks with `go build`.
+
+In every check directory there is an example configuration file called `local.toml.example`, the checks reads a
+file named `local.toml` by default, so by copying that file you can do a local test of your code before you commit.
+
+Full example to build and test run one check:
+```
+cd cmd/vulcan-drupal
+go build
+cp local.toml.example local.toml
+./vulcan-drupal
```

Comment:
I believe for the local checks to work with the provided file, the `-t` flag is required. I think this line should be:
```./vulcan-drupal -t```

Reply:
info added in new commit, thanks for feedback :)